src/toil_scripts/tools/preprocessing.py

```diff
@@ -167,13 +319,15 @@ def run_realigner_target_creator(job, cores, bam, bai, ref, ref_dict, fai, phase
         parameters.extend(['-U', 'ALLOW_SEQ_DICT_INCOMPATIBILITY'])
     docker_call(tool='quay.io/ucsc_cgl/gatk:3.5--dba6dae49156168a909c43330350c6161dc7ecc2',
                 inputs=inputs,
-                outputs={'sample.intervals': None},
-                work_dir=work_dir, parameters=parameters, env=dict(JAVA_OPTS='-Xmx{}'.format(mem)))
+                outputs={'sample.intervals': None}, mock=mock,
+                work_dir=work_dir, parameters=parameters,
+                env=dict(_JAVA_OPTIONS='-Djava.io.tmpdir=/data/ -Xmx{}'.format(mem)))
     # Write to fileStore
     return job.fileStore.writeGlobalFile(os.path.join(work_dir, 'sample.intervals'))
 
 
-def run_indel_realignment(job, intervals, bam, bai, ref, ref_dict, fai, phase, mills, mem, unsafe=False):
+def run_indel_realignment(job, intervals, bam, bai, ref, ref_dict, fai, phase, mills, mem,
+                          unsafe=False, mock=False):
```

Comment:
Document `mock`. Split params out to one per line unless there is a logical grouping.